sdk/python/test.py

```
@@ -1,17 +1,96 @@
-from .ragflow_sdk import RAGFlow
+"""
+You may use this script to test features related to the Python SDK.
+Execution instructions are as follows:
+>>> python test.py --api-key yourkeys --base-url YourBaseUrl --agent-id YourAgentID
 
-rag_object = RAGFlow(api_key="ragflow-FDfRECsXDRagsKPxb_EfZdDPcmngavSgYEzbU_Blgq4", base_url="http://localhost:9222")
-assistant = rag_object.get_agent("b0bc46e43dfc11f1b4ff84ba59bc54d9")
-session = assistant.create_session()
+You must provide api-key and agent-id via command‑line arguments or environment variables.
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)